src/lib/io/include/sol-fs-storage.h

```diff
@@ -106,6 +106,14 @@ int sol_fs_read_raw(const char *name, struct sol_buffer *buffer);
         return -EINVAL; \
     }
 
+/**
+ * @brief Read an uint8_t from storage
+ *
+ * @param name name of property. It will look for a file on filesystem with
+ * this name
+ * @param value value the uint8_t read from the storage
```

Comment:
You're repeating the value two times in this sentence, same problem occours in the other functions.
After this is fixed we can merge it.